src/revivalpmmp/pureentities/entity/monster/walking/Wolf.php

```diff
@@ -401,12 +404,36 @@ public function isFriendly(): bool {
     }
 
     /**
+     * Sets the collar color when tamed
+     *
+     * @param $collarColor
+     */
+    public function setCollarColor($collarColor) {
+        if ($this->isTamed()) {
+            $this->namedtag->CollarColor = new ByteTag(self::NBT_KEY_COLLAR_COLOR, $collarColor); // set collar color
```

Comment:
@kniffo80 Are you sure is correct that? Collar it's different in single player...

![image](https://cloud.githubusercontent.com/assets/13573011/23210583/f17578ba-f906-11e6-8266-b9768d76b41c.png)

Watch this code: https://github.com/Creeperface01/MinecraftSource/blob/master/src/net/minecraft/entity/passive/EntityWolf.java#L56